.travis.yml

```diff
@@ -160,6 +160,35 @@ jobs:
         - PLAT=i686
         - LATEST="true"
 
+    - name: "3.6 Xenial 64-bit PyPy"
+      os: linux
+      env:
+        - MB_PYTHON_VERSION=pypy3.6-7.3.1
```

Comment:
Does this need to be a full `pypy3.x-a.b.c` version? Can it be more generic?

Reply:
Maybe. The wheel that is created is called [Pillow-7.2.0.dev0-pp36-pypy36_pp73-manylinux2010_x86_64.whl](https://travis-ci.org/github/python-pillow/pillow-wheels/jobs/693175796#L5026), where the ABI tag is `pypy36_pp73`. So we might be able to get away with `pypy3.6-7.3` but I seem to recall trying that for NumPy and it didn't work. While PyPy 3.6-7.3.0 is ABI compatible with 7.3.1, some of the python code to handle python tags changed, and I think multibuild cannot use PyPy 3.6-7.3.0 to build wheels.